fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -167,6 +167,16 @@ protected static Counter getCounter(@Nonnull Map<Event, Counter> counters, @Nonn
          * @return the user-visible title
          */
         String title();
+
+        /**
+         * Get the key of this event for logging. This should be used with
+         * KeyValueLogMessages and other key-value based logging systems to
+         * log the values from instrumented events. These keys are not
+         * expected to change frequently.
```

Comment:
It may be worth noting that though they shouldn't change too frequently, they may change outside of any minor version change (or something) and so the value of these should not be relied upon outside of producing logs. Or maybe that's overkill.